views/common.twig

```diff
@@ -503,6 +503,8 @@
                 layer: "{{ "Layer"|trans }}",
                 dataSlot: "{{ "Data Slot"|trans }}",
                 dataSlotHelpText: "{{ "When there are more than one of the same element for a widget you can set the slot for each element. For example with two of the same element you'd have data slot 1 and data slot 2. If 10 items were returned slot 1 would receive items 1,3,5,7,9 and slot 2 would receive items 2,4,6,8,19."|trans }}",
+                pinSlot: "{{ "Pin Slot"|trans }}",
+                pinSlotHelpText: "{{ "Explain this please!!!!."|trans }}",
```

Comment:
We ought to get this set before merging.

Reply:
Oh sure, totally forgot about it, thank you Dan!

Reply:
Pin Item?
Once a data item has been assigned to this slot it will be pinned and will not cycle with the other data slots.